test/v2/support/rescue.js

```diff
@@ -3,6 +3,12 @@ const { POST, Request } = require('../../../api/classes/Request')
 
 const HTTP_CREATED = 201
 
+/**
+ * Create a rescue payload
+ * @param auth authentication credentials
+ * @param r request
```

Comment:
`r` is actually the rescue details in the format expected by the `/rescue` endpoint

Reply:
Can we get out of the habit of using single character variable names? It's not going to make the code any faster, but it *will* make the code less readable, even with JSDoc.

Reply:
yeah - sorry :D

Reply:
YOU BETTER BE SORRY.

Reply:
🤗